packages/data-stores/src/plans/hooks/use-pricing-meta-for-grid-plans.ts

```diff
@@ -112,6 +113,7 @@ const usePricingMetaForGridPlans = ( {
 					originalPrice: Plans.PlanPricing[ 'originalPrice' ];
 					discountedPrice: Plans.PlanPricing[ 'discountedPrice' ];
 					currencyCode: Plans.PlanPricing[ 'currencyCode' ];
+					introOffer: Plans.PlanPricing[ 'introOffer' ];
```

Comment:
`Non-blocking / Nitpick:`

It might be worth clearly differentiating between `discountPrice` and `introOffer`, whether it be an inline comment here or elsewhere. I'm assuming that intro offers are also discounted prices, but for _new_ customers only?

Reply:
Intro offers are a separate entity with it's own set of rules and configuration. Here's some more info :)
PCYsg-Wr4-p2